docker/cla_frontend.ini

```diff
@@ -10,3 +10,5 @@ logformat={"process_name": "uwsgi", "timestamp_msec": %(tmsecs), "method": "%(me
 post-buffering = 1
 buffer-size=32768
 post-buffering-bufsize=32768
+http-workers=5
```

Comment:
Out of interest, what was this defaulting to previously?

Reply:
It was defaulting to 1 http worker, the number of http workers is displayed when uwsgi starts